restapi.js

```diff
@@ -0,0 +1,196 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+const mongoose = require('mongoose')
+
+
+const {User, Transaction} = require('./model')
+
+const app = express()
+
+app.use(bodyParser.json())
+
+const port = 4500;
+
+const DBUri = "mongodb://localhost:27017/Users"
+
+mongoose.connect(DBUri)
+.then(() => {
+    console.log(`Database is connected`)
+})
+.catch(() => {
+    console.log(`Database is not connected`)
+});
+
+
+
+// Create an account
+/* e.g;
+{
+    "accountNumber": 3678299923,
+    "firstName": "Atinuke",
+    "lastName": "Gbolagade"
+}
+*/
+app.post('/createaccount/', async (req, res) => {
+    try {
```

Comment:
there is no validation here, yes the mongoose will return an error if I put numbers in the first and last name, but you should handle that error. also if I pass an empty string as the request there is also no validation.

Reply:
Hi @IGNORAMUS1 I tried this multiple times but it's still the same, looking at the code the only thing here is the unique field you added which will technically work but will cause the server to break, there is a better way to go about this by simply doing a DB lookup if the exact first and last name exists. You can go ahead and implement this yourself as I will be merging this.